test/index.js

```diff
@@ -1,4 +1,4 @@
 require('require-all')({
   dirname: __dirname,
-  excludeDirs: process.env.CI ? /^\./ : /^\.|benchmark/
+  excludeDirs: /^\.|benchmark/
```

Comment:
Do not run benchmark tests with the rest of the unit tests because it requires different runtime (node instead of babel-node).